packages/lodestar/src/chain/forkChoice/interface.ts

```diff
@@ -21,11 +23,11 @@ export interface ILMDGHOST {
 /*
  * Info of Block and Chain for forkchoice
  */
-export interface BlockChainInfo {
+export interface BlockHeadInfo {
```

Comment:
Somewhat unrelated to the original PR, I'd like to expose the whole block-tree with these objects
eg: `getAtSlot(slot: Slot): BlockHeadInfo` (which would navigate from the head, backwards up the chain to return the info at the slot.)
In that case, perhaps we should have a slightly more neutral naming, eg: `ChainInfo` or `ChainSummary`.
Also renaming `xxxRootBuf` to `xxxRoot` for more standard naming.